TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/ItemsListFragment.kt

```diff
@@ -16,31 +17,37 @@ import android.widget.ListView
 import androidx.fragment.app.Fragment
 import com.google.android.material.floatingactionbutton.FloatingActionButton
 import com.microsoft.device.display.samples.twonote.model.DataProvider
+import com.microsoft.device.display.samples.twonote.model.DirEntry
+import com.microsoft.device.display.samples.twonote.model.Inode
 import com.microsoft.device.display.samples.twonote.model.Note
 import com.microsoft.device.dualscreen.layout.ScreenHelper
+import java.io.*
```

Comment:
wildcard import (i think there's a setting in android studio where you can stop it from autocorrecting to * if you go to editor or coding style)